extensions/modules/widget-utils/notes/feature-hit_test.md

```diff
@@ -0,0 +1,4 @@
+# Path hit testing
+
+* A function `hitTestPath` has been added providing hit testing of a
```

Comment:
Is this what you meant @peter-b ? I still haven't quite worked out the note format for LCB yet